packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -374,7 +373,14 @@ export class ApolloServerBase<Request = RequestInit> {
       schema: this.schema,
       extensions: this.extensions,
       context,
-      // allow overrides from options
+      // Allow overrides from options. Be explicit about a couple of them to
```

Comment:
This is weird but I dunno if there's anything better.

Reply:
Seems odd, though I don't think it's worth destructuring to assign the two variables [here](https://github.com/apollographql/apollo-server/pull/1126/files#diff-3467ccc78fbd796513acd747ff2110a5L85)